covid19-notebooks/covid19-notebook-etl-run.sh

```diff
@@ -22,6 +21,9 @@ for file in "${NOTEBOOKS[@]}"; do
   jupyter nbconvert --to notebook --inplace --execute "$file"
 done
 
+echo "Running SIR-based Bayesian model script..."
+python3 covid19-sir-bayes-model/sir_model.py
```

Comment:
looking at the Dockerfile, i think you might need `sir_model.py` here instead of `covid19-sir-bayes-model/sir_model.py` 🤔 does this work?

Reply:
or, right, will change it 👍